opal/runtime/opal_params_core.c

```diff
@@ -89,10 +89,11 @@ static bool opal_register_util_done = false;
 
 static char *opal_var_dump_color_string = NULL;
 
-static char *opal_var_dump_color_keys[OPAL_VAR_DUMP_COLOR_KEY_COUNT] = {
+static char *opal_var_dump_color_keys[OPAL_VAR_DUMP_COLOR_KEY_COUNT+1] = {
     [OPAL_VAR_DUMP_COLOR_VAR_NAME] = "name",
     [OPAL_VAR_DUMP_COLOR_VAR_VALUE] = "value",
-    [OPAL_VAR_DUMP_COLOR_VALID_VALUES] = "valid_values"
+    [OPAL_VAR_DUMP_COLOR_VALID_VALUES] = "valid_values",
+    [OPAL_VAR_DUMP_COLOR_KEY_COUNT] = NULL
```

Comment:
as the array is static this isn't required

Reply:
...but a lot less confusing 😄

Reply:
I don't have a strong opinion about assigning NULL to the last entry or not.  I think the `+1` of the size is the real fix.  I am *pretty sure* that C99 will initialize any uninitialized values to NULL, but there's certainly no harm in making it explicit.

Indeed, if I were reading the code with no background info, I might have to go check that in this specific case -- a static array with partially-initialized elements -- will C99 will 0-ify the other elements?  So perhaps having it be explicit is not a bad thing: it avoids forcing those reading the code to go look up what C99 does in a corner case.